lib/src/style/QlementineStyle.cpp

```diff
@@ -861,8 +861,31 @@ void QlementineStyle::drawPrimitive(PrimitiveElement pe, const QStyleOption* opt
         const auto focus =
           widgetHasFocus && selection == SelectionState::Selected ? FocusState::Focused : FocusState::NotFocused;
         const auto active = getActiveState(itemState);
-        const auto& color = listItemBackgroundColor(mouse, selection, focus, active, optItem->index, w);
-        p->fillRect(rect, color);
+        if (const QStyleOptionViewItem *vopt = qstyleoption_cast<const QStyleOptionViewItem*>(opt)) {
+            QPalette::ColorGroup cg = (w ? w->isEnabled() : (vopt->state & QStyle::State_Enabled))
+                                      ? QPalette::Normal : QPalette::Disabled;
+            if (cg == QPalette::Normal && !(vopt->state & QStyle::State_Active))
+                cg = QPalette::Inactive;
```

Comment:
Please use brackets 😊

```suggestion
            if (cg == QPalette::Normal && !(vopt->state & QStyle::State_Active)) {
                cg = QPalette::Inactive;
            }
```